Utilities/DataLoading/EdFi.LoadTools.Test/LogConfiguration.cs

```diff
@@ -4,15 +4,15 @@
 // See the LICENSE and NOTICES files in the project root for more information.
  
 using log4net.Config;
-using Microsoft.VisualStudio.TestTools.UnitTesting;
+using NUnit.Framework;
 
 namespace EdFi.LoadTools.Test
 {
-    [TestClass]
+    [SetUpFixture]
```

Comment:
I think this may be wrong, please refer to https://github.com/nunit/docs/wiki/SetUp-and-TearDown for more infomation on how to use setup and tear down.

Reply:
SetupFixure is still supported  and used to call onetimesetup method first in that namespace before any test method execution which is needed for Log4net .  Please refer https://github.com/nunit/docs/wiki/SetUpFixture-Attribute